### PR TITLE
Tickets/preops 5546

### PR DESCRIPTION
**Pull request by ChristinaAdair**
HI Jeff, I am having a small challenge with previewing changes made to the delegate assembly schedule page. This pull request is to help view the changes.

### CODE REVIEW

File: docs/delegate-resources/assemblies-schedule.inc [hunk 1]
```diff
@@ -33,68 +33,60 @@
...
+   * - 2024-10-24
      - drop-in
-     - Q&A with Rubin staff, co-working time
-   * - 2024-04-04
+     - team members available for support
+   * - 2024-10-31
      - tutorial
-     - `Phase curve analysis in the Portal Aspect <https://community.lsst.org/t/rubin-science-assembly-thu-4-april-2024-at-9-am-pdt-pd0-3-phase-curve-fit-analysis-in-the-portal-aspect/8429>`_ (Yumi Choi)
-   * - 2024-04-11
+     - Aaron Meisner: DP0.2 Command line tutorial 02 (tentative)
```
Comment (jeffcarlin) [inline docs/delegate-resources/assemblies-schedule.inc]: This title has been updated on the Confluence page. (Should be "Butler From the Command Line")

File: docs/delegate-resources/assemblies-schedule.inc [hunk 2]
```diff
@@ -33,68 +33,60 @@
...
+   * - 2024-11-07
      - drop-in
-     - Q&A with Rubin staff, co-working time
-   * - 2024-04-18
+     - team members available for supporttopic
```
Comment (jeffcarlin) [inline docs/delegate-resources/assemblies-schedule.inc]: "supporttopic" --> "support"?

File: docs/delegate-resources/assemblies-schedule.inc [hunk 3]
```diff
@@ -33,68 +33,60 @@
...
+   * - 2024-12-19
+     - CANCELLED
+     - (winter break)
+   * - 2024-12-26
+     - CANCELLED
+     - (winter break)
+   * - 2025-01-02
+     - CANCELLED
+     - (winter break)
+   * - 2025-01-09
```
Comment (jeffcarlin) [inline docs/delegate-resources/assemblies-schedule.inc:78]: Please add "Yumi Choi: how to join user-uploaded tables in RSP" on this date

**Review by jeffcarlin — APPROVED** [review]
A couple of comments (we updated the Confluence schedule after you had already started working on this). Then it's ready to go!

File: docs/delegate-resources/assemblies-schedule.inc (hunk 2 above)
Comment (ChristinaAdair) [reply docs/delegate-resources/assemblies-schedule.inc]: fixed

File: docs/delegate-resources/assemblies-schedule.inc (hunk 3 above)
Comment (ChristinaAdair) [reply docs/delegate-resources/assemblies-schedule.inc:78]: added

**Pull request merged**